dataproc_jupyter_plugin/controllers/composer.py

```diff
@@ -32,7 +32,10 @@ async def get(self):
                     await credentials.get_cached(), self.log, client_session
                 )
                 environments = await client.list_environments()
-                self.finish(json.dumps(environments, default=lambda x: x.dict()))
+                if isinstance(environments, list):
+                    self.finish(json.dumps(environments, default=lambda x: x.dict()))
+                else:
+                    self.finish(environments)
```

Comment:
This change is not correct, we should continue to use the `json.dumps` method to ensure the written output is JSON.

What is the effect you are trying to achieve here?

The only practical difference between this and what we have now is that *if* the `environments` value is a dictionary, then the `Content-Type` header will be set to `application/json`.

Is that what you are trying to do? If so, then we should set that header regardless of the type of `environments`.

I.E. merely call `self.set_header("Content-Type", "application/json")` prior to calling `self.finish(...)`

Reply:
Added Content-Type header will be set to 'application/json' for composer list environments.